src/main/java/mods/eln/node/AutoAcceptInventoryProxy.kt

```diff
@@ -26,8 +27,10 @@ class AutoAcceptInventoryProxy(val inventory: IInventory) {
             if (inventory.getStackInSlot(index) == null) {
                 GenericItemUsingDamageDescriptor.getDescriptor(itemStack)?.let { desc ->
                     if (acceptedItems.any { it.isAssignableFrom(desc.javaClass) }) {
+                        val newItemStack = desc.newItemStack()
+                        (desc as? BatteryItem)?.let { it.setEnergy(newItemStack, it.getEnergy(itemStack)) }
```

Comment:
If you use apply instead of let, you get this instead of it. (And can usually skip the this.)

That's a matter of taste. This is fine.